consensus/polybft/validator_account.go

```diff
@@ -14,83 +15,100 @@ import (
 	"github.com/umbracle/fastrlp"
 )
 
-// ValidatorAccount represents a validator from the validator set
-type ValidatorAccount struct {
-	Address types.Address
-	BlsKey  *bls.PublicKey
+// ValidatorMetadata represents a validator metadata (its public identity)
+type ValidatorMetadata struct {
```

Comment:
Agreed, although this file consists of `ValidatorMetadata` and `AccountSet`, so not sure what is the proper name for it. :thinking: 

Btw, I think that we should reorganize code differently, because `polybft` package became pretty large. As part of such initiative, it is worth figuring out if whether we should split `AccountSet` and `ValidatorMetadata` to a separate files.
Although this is not a high priority at the moment I'd say.

Reply:
Fixed [3e02cbf](https://github.com/0xPolygon/polygon-edge/pull/865/commits/3e02cbfb20a8c1d93f82be235b8a8a751a36bdbc)